tests/test_validate_csvs.py

```diff
@@ -5,7 +5,7 @@
 def test_input_example_matrices():
     directory = "parameters_example/"
 
-    csv_files = [f for f in os.listdir(directory) if f.endswith('.csv')]
+    csv_files = [f for f in os.listdir(directory) if (f.endswith('.csv') and (f.startswith('lockdown') or f.startswith('actions')))]
     assert csv_files, "No CSV files found in the directory."
```

Comment:
It has to do with the fact that the naming and ordering of the variables must be the same across files, because these are then used in subsequent matrix multiplications in the model. There have been already problems in the past, i.e., LA changing the order/naming of variables not consistently. The code won't break but the meaning of the multiplication is altered, so that's why we need to check. Good to know that this is not clear at all -- I'll add a disclaimer/link the right issue.

Reply:
I agree, a comment on this would help.

Another nit-picky thing (feel free to ignore): if this is to make sure that the environment as provided by the user is fine, this should raise an exception (ie. `if not ...: raise ValueError(...)`.
I only use `assert` to check on assumptions that I have made inside the code, not on input data.

Reply:
ah interesting, thanks! I'll add it right away